diagnostic.md

```diff
@@ -3,36 +3,44 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Low bandwith environments with single page applications.
+You can control state with API only development.
+Can do away with jquery.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+The templating system is handlebars. You load templates and components based on URLs.
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+View
+Model
+View Model
```

Comment:
Views are a layer of 1.0 apps.
All of the layers are:
Router
Routes
Models
Templates
Components
Adapters
Controllers (1.0)
Views(1.0)

.5/1